models/dish/dishModel.js

```diff
@@ -0,0 +1,77 @@
+import { Schema, model } from 'mongoose';
+
+const dishSchema = new Schema(
+  {
+    name: {
+      type: String,
+      required: [true, 'Name of the dish is required'],
+      unique: true,
+    },
+    owner: {
+      type: Schema.Types.ObjectId,
+      ref: 'chef',
+      required: [true, 'Owner (chef) ID is required'],
+    },
+    image: {
+      type: String,
+      required: [true, 'Image URL is required'],
+    },
+    description: {
+      type: String,
+      required: [true, 'Description of the dish is required'],
+    },
+    ingredients: {
+      type: Schema.Types.ObjectId,
+      ref: 'ingredient',
+      required: [true, 'Ingredient ID is required'],
+    },
+    price: {
+      type: Number,
+      required: [true, 'Price of the dish is required'],
+    },
+    isVegan: {
```

Comment:
maybe it's better to use a set of prepared values: vegan, vegeterian, pescaterian etc.

Reply:
also can do multiple choice for the new approach if decide to use it

Reply:
Maybe in future. To simplify it we decided to leave only this option. But we will think about it.